src/ClusterBootstrap/template/cloud-config/cloud_init_worker.txt.template

```diff
@@ -11,7 +11,6 @@ runcmd:
  - echo "export KUBE_LABELS=\"{{cnf['kube_labels'] | join(',')}}\"" >> boot.env
  - echo "export ETCD_ENDPOINTS='{{cnf["etcd_endpoints"]}}'" >> boot.env
  - echo "export KUBE_API_SERVER='{{cnf["api_servers"]}}'" >> boot.env
- - echo "export GPU_TYPE='{{cnf["gpu_type"]}}'" >> boot.env
```

Comment:
Is this environment variable used when setting node label `gpuType=<gpu_type_val>`?

Reply:
not anymore, for labels, worker machines would use cloud-init files that is related to sku.
for RestfulAPI/config.yaml, I also removed the dependency